### ISSUE TITLE
Translation Contributions Wanted 🙋🙏

### ISSUE THREAD

**Issue by iamkun** [dayjs/dayjs-website#5]
Day.js users come from all around the world, so I will happily accept translations from other languages for this project. And your work will affect thousands of daily users of Day.js.

We are now hosting the whole translation process on https://crowdin.com/project/dayjs

Feel free to leave a comment if you want to join us. Thanks.

-------
Note: We have some previous translation here that might help  https://github.com/iamkun/dayjs/tree/dev/docs.

**Comment by 53able** [comment]
Hi,
Can I help with the Japanese translation?

**Comment by 53able** [comment]
@iamkun 
Hi,
Can I help with the Japanese translation?

**Comment by iamkun** [comment]
@53able That would be nice.

**Comment by 53able** [comment]
@iamkun Consider it done🙋‍♂️

**Comment by iamkun** [comment]
@53able Cool. Please check if you could get access to the Japanese part here https://crowdin.com/project/dayjs. And feel free to ask me anything if needed.

**Comment by CreativeRusBear** [comment]
@iamkun, hi 
I also can help you with translation on russian language

**Comment by iamkun** [comment]
@CreativeRusBear Thanks, you can join our group your self https://crwd.in/dayjs, so that I can invite you as a proofreader. 

Moreover, we can talk live here https://gitter.im/iamkun/dayjs.

**Comment by iamkun** [comment]
Russian translation progress is over 90%. 

We've enabled the entrance on our doc website on the top right corner.

Thanks @CreativeRusBear.

Cheers! 🎉🎉🎉🎉🎉

https://day.js.org/ru/

**Comment by CreativeRusBear** [comment]
@iamkun I'm glad to help you 😉

**Comment by W3ND31** [comment]
The Brazilian Portuguese translation still active?

**Comment by iamkun** [comment]
> The Brazilian Portuguese translation still active?

@W3ND31 Seems not anymore. Would you like to continue it?

**Comment by W3ND31** [comment]
> > The Brazilian Portuguese translation still active?
> 
> @W3ND31 Seems not anymore. Would you like to continue it?

Yes! I would like to help the project, even that is a small help compared with the whole project :)

**Comment by iamkun** [comment]
@W3ND31 Cheers! You can join our group here https://crowdin.com/project/dayjs

And note, we have some previous translation here that might help https://github.com/iamkun/dayjs/tree/dev/docs.

**Comment by bhatvikrant** [comment]
Hi, can i help in translating to `Hindi` language

**Comment by iamkun** [comment]
@bhatvikrant That would be nice. You can join our group here https://crowdin.com/project/dayjs

**Comment by bhatvikrant** [comment]
@iamkun how does it work? I just update the README and push it here `https://crowdin.com/project/dayjs` ?

**Comment by iamkun** [comment]
@bhatvikrant All you have to do is to log in to Crowdin via your GitHub account and click the language that you want to contribute.

Then use Crowdin's online IDE to submit your translation in each file of the document.

**Comment by bhatvikrant** [comment]
Thanks, got it

**Comment by doliG** [comment]
Hello @iamkun , can I help for French translation ?

**Comment by iamkun** [comment]
@doliG Great! You can join our group here https://crowdin.com/project/dayjs

And note, we have some previous French translation here that might help https://github.com/iamkun/dayjs/tree/dev/docs.

**Comment by wonderlandpark** [comment]
Hi @iamkun, can I help for Korean translation?

**Comment by iamkun** [comment]
@wonderlandpark Great! You can find the Korean section here https://crowdin.com/project/dayjs

And note, we have some previous Korean translation here that might help https://github.com/iamkun/dayjs/tree/dev/docs

**Comment by Alwayslazydog** [comment]
hi, can i help for japanese translation?

**Comment by Alwayslazydog** [comment]
halo @iamkun, can i help for japanese translation

**Comment by iamkun** [comment]
@Alwayslazydog Thanks, feel free to visit https://crowdin.com/project/dayjs and join us there directly.

**Comment by doliG** [comment]
Hey guys,

I'm currently working on French translations, and I'm struggling with a simple question: should I also translate sections titles?

Most titles are "methods" names (or they look alike), and I'm not sure it's relevant to do it. I mean, if a user go to the translated docs, he'll probably search for a method in English, but wish the description to be in his language.

So I'm not sure what to do. Should I translate it, or should I keep all the titles in English? 🤔 

What's your opinion?

**Comment by iamkun** [comment]
@doliG I did not translate the title of the Chinese translation. Cause most of the title is also the API name which is more familiar to our user and accurate. But feel free to make some adjustments within your own language.

**Comment by honederr82** [comment]
Hi, I could help with German translation!

**Comment by iamkun** [comment]
@honederr82 Hi, feel free to join us here https://crowdin.com/project/dayjs/de#

**Comment by munkhorgil** [comment]
Hello @iamkun, I'd like to translate into Mongolian, I already joined the Crowdin.

**Comment by iamkun** [comment]
@munkhorgil Thanks, join us Mongolian translation here https://crowdin.com/project/dayjs/mn

**Comment by zhulien-ivanov** [comment]
I can provide translations in Bulgarian.

**Comment by Kingong** [comment]
Can I Help For Chinese Translation？
